zone/command.cpp

```diff
@@ -9080,709 +9080,865 @@ void command_npcedit(Client *c, const Seperator *sep)
-		std::string query = StringFormat("UPDATE npc_types SET special_abilities = '%s' WHERE id = %i",  sep->argplus[2],npcTypeID);
+        c->Message(Chat::Yellow, fmt::format("NPC ID {} is now using the following Special Abilities '{}'.", npc_id, sep->arg[2]).c_str());
+		std::string query = fmt::format("UPDATE npc_types SET special_abilities = '{}' WHERE id = {}", sep->arg[2], npc_id);
 		content_db.QueryDatabase(query);
 		return;
 	}
 
-	if (strcasecmp(sep->arg[1], "spell") == 0) {
-        c->Message(Chat::Yellow,"NPCID %u now uses spell list %i",  npcTypeID, atoi(sep->argplus[2]));
-		std::string query = StringFormat("UPDATE npc_types SET npc_spells_id = %i WHERE id = %i",  atoi(sep->argplus[2]),npcTypeID);
+	if (strcasecmp(sep->arg[1], "aggroradius") == 0) {
+        c->Message(Chat::Yellow, fmt::format("NPC ID {} now has an Aggro Radius of {}.", npc_id, atoi(sep->arg[2])).c_str());
+		std::string query = fmt::format("UPDATE npc_types SET aggroradius = {} WHERE id = {}", atoi(sep->arg[2]), npc_id);
 		content_db.QueryDatabase(query);
 		return;
 	}
 
-	if (strcasecmp(sep->arg[1], "faction") == 0) {
-        c->Message(Chat::Yellow,"NPCID %u is now faction %i",  npcTypeID, atoi(sep->argplus[2]));
-		std::string query = StringFormat("UPDATE npc_types SET npc_faction_id = %i WHERE id = %i",  atoi(sep->argplus[2]),npcTypeID);
+	if (strcasecmp(sep->arg[1], "assistradius") == 0) {
+        c->Message(Chat::Yellow, fmt::format("NPC ID {} now has an Assist Radius of {}", npc_id, atoi(sep->arg[2])).c_str());
+		std::string query = fmt::format("UPDATE npc_types SET assistradius = {} WHERE id = {}", atoi(sep->arg[2]), npc_id);
 		content_db.QueryDatabase(query);
 		return;
 	}
 
-	if (strcasecmp(sep->arg[1], "damage") == 0) {
-        c->Message(Chat::Yellow,"NPCID %u now hits from %i to %i",  npcTypeID, atoi(sep->arg[2]), atoi(sep->arg[3]));
-		std::string query = StringFormat("UPDATE npc_types SET mindmg = %i, maxdmg = %i WHERE id = %i",  atoi(sep->arg[2]), atoi(sep->arg[3]), npcTypeID);
+	if (strcasecmp(sep->arg[1], "featuresave") == 0) {
+        c->Message(Chat::Yellow, fmt::format("NPC ID {} saved with all current facial feature settings.", npc_id).c_str());
+        Mob* target = c->GetTarget();
+		std::string query = fmt::format(
+			"UPDATE npc_types "
+			"SET luclin_haircolor = {}, luclin_beardcolor = {}, "
+			"luclin_hairstyle = {}, luclin_beard = {}, "
+			"face = {}, drakkin_heritage = {}, "
+			"drakkin_tattoo = {}, drakkin_details = {} "
+			"WHERE id = {}",
+			target->GetHairColor(), target->GetBeardColor(),
+			target->GetHairStyle(), target->GetBeard(),
+			target->GetLuclinFace(), target->GetDrakkinHeritage(),
+			target->GetDrakkinTattoo(), target->GetDrakkinDetails(),
+			npc_id
+		);
+		content_db.QueryDatabase(query);
+		return;
+	}
+
+	if (strcasecmp(sep->arg[1], "armortint_id") == 0) {
+        c->Message(Chat::Yellow, fmt::format("NPC ID {} is now using Armor Tint ID {}.", npc_id, atoi(sep->arg[2])).c_str());
```

Comment:
This shouldn't need a `c_str()` but I'm not asking you to change it

Reply:
Not sure why I wouldn't need .c_str(), fmt::format() returns a string and Message needs a const char*.